src/Repositories/Concerns/Testing.php

```diff
@@ -15,33 +14,41 @@
  */
 trait Testing
 {
-    public static function route(string $path = null, array $query = []): Stringable
-    {
-        if (str($path)->startsWith('/')) {
-            $path = str($path)->replaceFirst('/', '')->toString();
+    public static function route(
+        string|Model $path = null,
+        Action $action = null,
```

Comment:
Having `Action` as a second argument, will be a breaking change for many projects. You can have it as a third argument eventually?